Policies/github-community-guidelines.md

```diff
@@ -48,23 +48,35 @@ Of course, you can always contact us to {% data variables.contact.report_abuse %
+- #### Gratuitously violent content*
+   Don’t post violent images, text, or other content without reasonable context or warnings. While it's often okay to include violent content in video games, news reports, and descriptions of historical events, we do not allow violent content that is posted indiscriminately, or that is posted in a way that makes it difficult for other users to avoid (such as a profile avatar or an issue comment). A clear warning or disclaimer in other contexts helps users make an educated decision as to whether or not they want to engage with such content.
 
-* **Active malware or exploits** - Being part of a community includes not taking advantage of other members of the community. We do not allow anyone to use our platform for exploit delivery, such as using GitHub as a means to deliver malicious executables, or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Note, however, that we do not prohibit the posting of source code which could be used to develop malware or exploits, as the publication and distribution of such source code has educational value and provides a net benefit to the security community.
+- #### Misinformation and disinformation
+   You may not post content that presents a distorted view of reality, whether it is inaccurate or false (misinformation) or is intentionally deceptive (disinformation) because such content is likely to result in harm to the public or to interfere with fair and equal opportunities for all to participate in public life. For example, we do not allow content that may put the well-being of groups of people at risk or limit their ability to take part in a free and open society. We encourage active participation in the expression of ideas, perspectives, and experiences and may not be in a position to dispute personal accounts or observations. We generally allow parody and satire that is in line with our Acceptable Use Polices, and we consider context to be important in how information is received and understood; therefore, it may be appropriate to clarify your intentions via disclaimers or other means, as well as the source(s) of your information.
```

Comment:
How does this affect artistic projects that distort reality on purpose for editorial comment?

Reply:
Thanks @aredridel, we've just made a [small edit to clarify](https://github.com/github/site-policy/pull/310/commits/1f55cf494f935c525597ae6da6529c6dc6b4c7cb). To your question:

Acceptable Use Policies restrict content that

> is or contains false, inaccurate, or intentionally deceptive information that is likely to adversely affect the public interest (including health, safety, election integrity, and civic participation);

Our Community Guidelines further note

> We generally allow parody and satire that is in line with our Acceptable Use Polices, and we consider context to be important in how information is received and understood

This leaves room for "artistic projects that distort reality on purpose for editorial comment," provided they do not violate those standards.